Description: Update .egg-info to match what is produced by current Python
Forwarded: not-needed
Last-Update: 2019-12-03

--- sgmllib3k-1.0.0.orig/sgmllib3k.egg-info/PKG-INFO
+++ sgmllib3k-1.0.0/sgmllib3k.egg-info/PKG-INFO
@@ -1,4 +1,4 @@
-Metadata-Version: 1.0
+Metadata-Version: 1.1
 Name: sgmllib3k
 Version: 1.0.0
 Summary: Py3k port of sgmllib.
--- sgmllib3k-1.0.0.orig/sgmllib3k.egg-info/SOURCES.txt
+++ sgmllib3k-1.0.0/sgmllib3k.egg-info/SOURCES.txt
@@ -1,4 +1,5 @@
 README
+setup.cfg
 setup.py
 sgmllib.py
 sgmllib3k.egg-info/PKG-INFO
